.vib/wordpress/cypress/cypress/fixtures/posts.json

```diff
@@ -3,4 +3,4 @@
   "content": "Welcome to WordPress.",
   "commenter": "A WordPress Commenter",
   "comment": "This is a test comment."
```

Comment:
Same here, it should all go under the same entry

Reply:
Done!